decidim-core/lib/decidim/has_settings.rb

```diff
@@ -18,6 +18,14 @@ def settings=(data)
       self[:settings]["global"] = serialize_settings(settings_schema(:global), data)
     end
 
+    def current_settings
+      if participatory_space.allows_steps?
```

Comment:
I am not really sure but `allows_steps?` sounds weird to me 😂 . What about `allow_steps?` instead? I summon @mrcasals for grammar issues 😝